applications/asset_tracker_v2/src/modules/gps_module.c

```diff
@@ -402,13 +411,19 @@ static void on_all_states(struct gps_msg_data *msg)
 			SEND_ERROR(gps, GPS_EVT_ERROR_CODE, err);
 		}
 
-		state_set(STATE_INIT);
+		state_set(STATE_RUNNING);
```

Comment:
```suggestion
		state_set(STATE_INIT);
```

Reply:
I'm guessing this should be `STATE_INIT`? Or do we want to go to the running state in the case of receiving
`MODEM_EVT_INITIALIZED` and `APP_EVT_START` ?